import/index_java/src/org/vufind/index/ConfigManager.java

```diff
@@ -137,6 +137,8 @@ public Ini loadConfigFile(String filename)
             File configFile = null;
             try {
                 configFile = findConfigFile(filename);
+            } catch (IllegalStateException e) {
+                dieWithError("Illegal State VUFIND_HOME not set");
```

Comment:
Rather than assuming this is related to VUFIND_HOME, would it be better to pass through the existing message in e?